portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,12 @@
+package com.google.sps.data;
+
+public final class Comment {
```

Comment:
Use Project Lombok. Refer to Sai's PR posted on chat for more details

Reply:
Used Lombok annotations for auto-generated constructors.